doc/lua_api.md

```diff
@@ -9095,19 +9095,20 @@ Used by `minetest.register_node`, `minetest.register_craftitem`, and
     -- Otherwise should be name of node which the client immediately places
     -- upon digging. Server will always update with actual result shortly.
 
-    touch_interaction = {
-        -- Only affects touchscreen clients.
-        -- Defines the meaning of short and long taps with the item in hand.
-        -- The fields in this table have two valid values:
-        -- * "long_dig_short_place" (long tap  = dig, short tap = place)
-        -- * "short_dig_long_place" (short tap = dig, long tap  = place)
-        -- The field to be used is selected according to the current
-        -- `pointed_thing`.
-
-        pointed_nothing = "long_dig_short_place",
-        pointed_node    = "long_dig_short_place",
-        pointed_object  = "short_dig_long_place",
+    touch_interaction = <TouchInteractionMode> OR {
```

Comment:
i think the way we generally document this is by writing it twice?

Reply:
Both forms exist. There are 20 occasions of the `OR` form. The "write it twice" form is used in the particlespawner docs (and probably in more places, but I can't easily grep for it).

Do you want me to change it?

Reply:
No need then.